src/fontra/views/editor/edit-behavior.js

```diff
@@ -22,6 +22,8 @@ export class EditBehaviorFactory {
       point: pointSelection,
       component: componentSelection,
       anchor: anchorSelection,
+      guidelineGlyph: guidelineGlyphSelection,
+      guidelineFont: guidelineFontSelection,
```

Comment:
Let's either leave this out completely, or at least commented out.

Reply:
On second thought, please remove all references to font guidelines here, because I don't think EditBehavior can deal with them, in a very fundamental way: EditBehavior is tied to the glyph, and the glyph changes, and the glyph undo stack.

I'm a little uncomfortable that we have to add support for guidelines here in the first place, and actually the same for anchors, but I don't readily have an alternative to suggest. I will try to come up with better factoring later, but that's for after this PR.

Reply:
I removed all 'fontGuideline' comments from edit-behavior